internal/cli/atlas/datalake/delete.go

```diff
@@ -54,10 +54,8 @@ func DeleteBuilder() *cobra.Command {
 		Use:     "delete <name>",
 		Aliases: []string{"rm"},
 		Short:   "Remove a federated database instance from your project.",
-		Long: `To learn more about Atlas Data Federation (previously named Atlas Data Lake), see https://www.mongodb.com/docs/atlas/data-federation/overview/.
```

Comment:
removing roles for these?

Reply:
Yes, they were added to the dataLakes commands by mistake (should have been added to privateEndpoints-dataLakes commands). This PR corrected that. The dataLakes commands are some of the few not in the OpenAPI spec, so I need more info on them before I can add roles for them